app/code/Magento/Checkout/ViewModel/Cart.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Checkout\ViewModel;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Framework\View\Element\Context;
+use Magento\Store\Model\ScopeInterface;
+
+class Cart implements ArgumentInterface
+{
+    /**
+     * Config settings path to enable clear shopping cart button
+     */
+    private const XPATH_CONFIG_ENABLE_CLEAR_SHOPPING_CART = 'checkout/cart/enable_clear_shopping_cart';
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $_scopeConfig;
+
+    /**
+     * Constructor
+     *
+     * @param Context $context
+     */
+    public function __construct(
+        Context $context
+    ) {
+        $this->_scopeConfig = $context->getScopeConfig();
+    }
+
+    /**
+     * Check if clear shopping cart button is enabled
+     *
+     * @return bool
+     */
+    public function isClearShoppingCartEnabled()
```

Comment:
@joanhe @danmooney2 don't we need integration tests for this new php funtion?

Reply:
This line gets called twice already in MFTF and returns both permutations of values (true and false).  We do however need additional coverage for website scope.  It'd be better and faster to automate that in an integration test.

Reply:
Added integration test, see file below:
`dev/tests/integration/testsuite/Magento/Checkout/ViewModel/CartTest.php`

Reply:
The integration test looks good